src/features/device-state/routes/state-get-v3.ts

```diff
@@ -12,7 +12,6 @@ import {
 	getStateDelayingEmpty,
 	getStateEventAdditionalFields,
 	readTransaction,
-	serviceInstallFromImage,
```

Comment:
@Page- I think that you can also drop this as part of your major #5477 after this gets merged

Reply:
@thgreasi It would probably be better to first stop using this on state get v1 before droping this one, but yes, I am aware we need a major to remove this one :)